apps/shared-modules/mini-user-profile/components/user-follow/user-follow.component.ts

```diff
@@ -90,8 +91,19 @@ export class UserFollowComponent implements OnChanges, OnDestroy {
     );
   }
 
-  openDialog(ref: TemplateRef<any>) {
-    this.nbDialogService.open(ref);
+  // openDialog(ref: TemplateRef<any>) {
+  //   this.nbDialogService.open(ref);
+  //   this.gtm.dataLayerPushEvent('user-follow-initiate', { com_followee_id: this.user.id });
+  // }
```

Comment:
why are we comment this?

Reply:
I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier